docs/api-docs/provider-apis/tax/overview.md

```diff
@@ -58,19 +58,41 @@ Once your tax provider configuration is ready, we'll let you know via email. The
 | **Testing**                               |                                |                              |                                                                                                       |                                                   |
 | Partner test store domain(s)          | Required                       | Domain name                  | Share your test store domain(s) sow we can test prior to launching your tax provider.                 | `https://sampletax-test-store.mybigcommerce.com/` |
 
+### Sandbox tax provider configuration
+
+Sandbox tax provider configurations are nested within your production tax provider configuration, as such tax providers requiring a sandbox tax provider configuration should also provide the details for their production tax provider configuration.
+
+Additionally, when providing the details for a sandbox tax provider configuration, it's unncessary to specify the below properties because they are inherited from the parent production tax provider configuration (all other properties are still required):
+
+* Supported store(s), i.e. all stores or supported store hashes in the case tax provider is a private instance
+* Supported / unsupported countries
```

Comment:
We also don't need: 
* Tax provider visibility
* Merchant support email  
* Merchant support URL

Reply:
Updated with [latest commit](dea6ca188207b7ac2eae1415d97256124461e275).